native/mod_manager/mod_manager.c

```diff
@@ -2980,17 +3012,26 @@ static char *process_domain(request_rec *r, char **ptr, int *errtype, const char
 }
 
 /* XXX: move to mod_proxy_cluster as a provider ? */
-static void printproxy_stat(request_rec *r, int reduce_display, const proxy_worker_shared *proxystat)
+static void printproxy_stat(request_rec *r, int reduce_display, nodeinfo_t *node)
 {
-    char *status = NULL;
-    status = proxystat->status & PROXY_WORKER_NOT_USABLE_BITMAP ? "NOTOK" : "OK";
+    char *status;
+    proxy_worker_shared *proxystat = read_shared_by_node(r, node);
+    if (!proxystat) {
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, r->server, "printproxy_stat can't find worker");
```

Comment:
I don't want to be nitpicking, but could you please add `:` after `printproxy_stat` in the debug message?